src/openepd/model/epd.py

```diff
@@ -19,15 +19,19 @@
 from openepd.model.base import BaseDocumentFactory, OpenEpdDoctypes
 from openepd.model.common import Amount, Ingredient, WithAltIdsMixin, WithAttachmentsMixin
 from openepd.model.declaration import BaseDeclaration
-from openepd.model.org import Org, Plant
+from openepd.model.lcia import WithLciaMixin
+from openepd.model.org import Org, OrgRef, Plant
 from openepd.model.specs import Specs
 from openepd.model.versioning import OpenEpdVersions, Version
 
 
-class EpdV0(WithAttachmentsMixin, WithAltIdsMixin, BaseDeclaration):
-    """Represent an EPD."""
+class EpdPreviewV0(WithAttachmentsMixin, WithAltIdsMixin, BaseDeclaration):
```

Comment:
Add title, like "EPD (Preview)", "EPD (Full)", "EPD (Create Mode)"?

Reply:
Do you mean first line of the docstring?

Reply:
Figured out, explicit "title" can be used as a class-constructor kwarg.